call-runtime/README.md

```diff
@@ -13,7 +13,6 @@ To integrate this example into Substrate you need to adjust pallet contracts con
     …
     // `Everything` or anything that will allow for the `Balances::transfer` extrinsic.
     type CallFilter = frame_support::traits::Everything; 
-    type UnsafeUnstableInterface = ConstBool<true>;
```

Comment:
we should probably keep it and specify for version of pallet-contract < xxx

Reply:
mm I am pretty sure that a lot of people will run this against the contract-node that is only updated at every release

Reply:
I updated it yesterday